lab-ronoh/lib/task-crud.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+// const debug = require('debug')('ronito: task-crud');
+const mongoose = require('mongoose');
+mongoose.Promise= require('bluebird');
```

Comment:
After looking through lots of docs, I now think that this is exactly the right way to do this, so ignore my comment above.